Sources/Fullscreen/FavoriteAdsList/FavoriteAdsListView.swift

```diff
@@ -56,6 +58,9 @@ public class FavoriteAdsListView: UIView {
         didSet { tableHeaderView.sortingTitle = sortingTitle }
     }
 
+    public var commentActionTitle = ""
```

Comment:
Noticed this one now, but we should probably ask the delegate for the titles. Seeing as we need different ones when the ad has a comment, vs. when it doesn't.